android/src/com/mapswithme/util/Config.java

```diff
@@ -296,6 +297,16 @@ public static void setTransliteration(boolean value)
     nativeSetTransliteration(value);
   }
 
+  public static boolean isWunderLINQEnabled()
+  {
+    return getBool(KEY_PREF_WUNDERLINQ, false);
+  }
+
+  public static void setWunderLINQEnabled(boolean show)
+  {
+    setBool(KEY_PREF_WUNDERLINQ, show);
+  }
```

Comment:
Please move this setting in SharedProperitesUtils.java to store it in shared preferences.

Reply:
Removed preference altogether, no longer required.